### PR TITLE
add initial CR review checklists

### PR DESCRIPTION
**Pull request by loganmcdonald-noaa**
This pull request adds initial files for a PR review checklist inside our workflows. 

The action will be added in a subsequent PR.

### CODE REVIEW

File: .github/workflows/code-review-templates/code-review-full.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Code Review Checklist
+
+This is an automated comment on every pull request requiring a review. A checked item is either an assertion that I have tested this item or an indication that I have verified it does not apply to this pull request.
+
+## The Basics
+- [ ] Checks are passing
+- [ ] I read the code
+- [ ] I ran the code
+- [ ] (if applicable) Post deploy steps are run
+- [ ] (if applicable) I validated the change on the deployed version in <environment>
```
Comment (greg-does-weather) [inline .github/workflows/code-review-templates/code-review-full.md:10]: I don't think I understand these. If we're reviewing it, doesn't that mean it hasn't been deployed yet? Or is this in the case of temporary deploys (a la cloud.gov Pages)?

File: .github/workflows/code-review-templates/code-review-full.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Reliability
+- [ ] error handling exists for unusual or missing values
+- [ ] interactions with external systems are wrapped in try/except
+- [ ] functionality is tested with unit or integration tests
+- [ ] dependency updates in composer.json also got changed in composer-lock.json
+
+## Infrastructure
+- [ ] all changes are auditable and documented via a script
+- [ ] it is clear who can and should run the script
+- [ ] (if applicable) diagrams been updated or added in PlantUML
```
Comment (greg-does-weather) [inline .github/workflows/code-review-templates/code-review-full.md:30]:
```suggestion
- [ ] (if applicable) diagrams have been updated or added in PlantUML
```

File: .github/workflows/code-review-templates/code-review-full.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Accessibility
+- [ ] New pages have been added to cypress-axe file so that they will be tested with our automated accessibility testing
+- Meets WCAG 2.0 AA or 2.1 AA for Section 508 compliance
+    - [ ] Site is keyboard accessible. All interactions can be accessed with a keyboard
+    - [ ] Site is free of keyboard traps. The keyboard focus is never trapped in a loop
+    - [ ] All form inputs have explicit labels
+    - [ ] Form instructions are associated with inputs
+    - [ ] All relevant images use an img tag
+    - [ ] All images have alt attributes
```
Comment (greg-does-weather) [inline .github/workflows/code-review-templates/code-review-full.md:40]:
```suggestion
    - [ ] All images have appropriate alt attributes
```

**Review by greg-does-weather — APPROVED** [review]
These seem like a solid start to me!

File: .github/workflows/code-review-templates/code-review-full.md (hunk 1 above)
Comment (loganmcdonald-noaa) [reply .github/workflows/code-review-templates/code-review-full.md:10]: Yep! temporary

**Pull request merged**